### DISCUSSION TITLE
Is macvz still relevant now that Colima has out of the box support for vz and virtiofs?

### DISCUSSION THREAD

**Discussion by pelletencate** [mac-vz/macvz#25] [Q&A]
Is macvz still relevant now that Colima has out of the box support for vz and virtiofs?

If so, it's not completely clear from the  README as to how.

**Comment by balajiv113** [comment]
@pelletencate - Yes, the initial goal of this project was to provide vz support for lima. 
Since lima was also interested in this idea from macOS 13, i managed to push this to lima.

As of this project, am still not sure on the goal (Am exploring ideas). But, one thing that am sure of is macvz will not be a rewrite of lima. Either macvz will address a different usecase or will achieve this project
